### ISSUE TITLE
TableStyle's width is ignored when writing docx

### ISSUE THREAD

**Issue by anrikun** [PHPOffice/PHPWord#629]
Code below should write a 100% width table as stated in documentation but it doesn't work:
```php
$table = $section->addTable(array(
  'unit' => \PhpOffice\PhpWord\Style\Table::WIDTH_PERCENT,
  'width' => 100 * 50,
));
```

Only the following works:
```php
$table = $section->addTable();
$table->setWidth(100 * 50);
```

**Comment by B-Prod** [comment]
I can confirm this with the last version of PHPWord.

**Comment by tanc** [comment]
Also confirmed.

**Comment by ant-s** [comment]
Yes, this is the case. I'm not sure why you also need 100 * 50 for the width. It's possible that if the width is just set to a really large number then it just takes up 100%.

**Comment by newbro** [comment]
Just hit this bug as well.

**Comment by rebekahocampo** [comment]
Does anyone know the status of this bug? I just installed phpoffice/phpword a couple days ago, and while developing my code, I noticed this same thing is happening. Very Confusing. Thanks @ant-s  for the alternative.

**Comment by joao-vieira** [comment]
Thanks @ant-s ,  your answer resolved my issue:

I was having trouble uploading a file I had generated with [phpword](https://github.com/PHPOffice/PHPWord) that contained a table. By uploading this and using the [storeAs](https://laravel.com/docs/5.4/filesystem) method, the table lost its edges.

Now it work.

**Comment by icy2003** [comment]
me too……on "phpoffice/phpword": "^0.16.0" (composer.json)

**Comment by chrisribe** [comment]
Same issue, trying to use a style defined in my document template example "TBL1" but also specify the table width which must be set per table. Seems it is not part of the style in MsWord.

**Comment by arif98741** [comment]
// Here is my working code for setting width to 100%

    $table_style = new \PhpOffice\PhpWord\Style\Table;
    $table_style->setBorderColor('cccccc');
    $table_style->setBorderSize(1);
    $table_style->setUnit(TblWidth::PERCENT);
    $table_style->setWidth(100 * 50);

    // Add a section to the document.
    $section = $this->phpword->addSection();

    // Set up our table.
    $table = $section->addTable($table_style);
